tests/mocha/field_textinput_test.js

```diff
@@ -183,6 +183,7 @@ suite('Text Input Fields', function() {
         if (this.stub) {
           this.stub.restore();
         }
+        Blockly.WidgetDiv.testOnly_setDiv(null);
```

Comment:
We want this clearing of the global variable to happen for all tests, not just this suite.
I think it should go in `sharedTestTeardown`

Reply:
Done